source/content/guides/decoupled/drupal-nextjs-frontend-starters/03-caching.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Drupal + Next.js Frontend Starter for Front-End Sites
+subtitle: Caching Recommendations
+description: Learn about caching recommendations for your Drupal frontend starter kit.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/drupal-nextjs-frontend-starters/caching
+anchorid: caching
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+Cache settings are configured out-of-the-box for you, but there are circumstances in which you might need to make your own configurations, for example, your CMS is not on Pantheon but your frontend is on Pantheon.
+
+## How it Works
+
+<Partial file="decoupled-caching.md" />
+
+## Using the Starter Kit
+
+The `PantheonDrupalState` class from the `@pantheon-systems/drupal-kit` npm
+package includes an adapted fetch method that adds the `Fastly-Debug` header
+to each request sent to Drupal. Responses from Drupal contain the
+`Surrogate-Key` header. You can use these keys to instruct your frontend to purge content from a cache when the content in Drupal changes.
+
+### Ensure Headers are Set on Custom Routes
```

Comment:
I'm not exactly sure the right way to describe this, but I feel this section could make it clearer that all of the existing routes in the starter kits will already do this. You'll only need to follow this as you create new routes. It is kind of implied by the heading, but I feel like it could be missed.